### ISSUE TITLE
Accessibility : Keyboard focus outlines not visible

### ISSUE THREAD

**Issue by HarshalCapgemini** [xamarin/Xamarin.Forms#14850]
Accessibility for Android 

Keyboard focus outlines not visible. Keyboard focus indicator does not have at least 3:1 contrast ratio.
It's not possible to easily see the keyboard focus in the app when using a bluetooth keyboard because there is no focus outline and the focus indicator does not have at least 3:1 contrast ratio. 
Use an easy to see focus outline for all controls and make sure it has at least 3:1 contrast ratio. 

Is this supported in Xamarin Forms? Can we manually set focus?

Related to https://github.com/xamarin/Xamarin.Forms/issues/9999

**Comment by rachelkang** [comment]
Hi, @HarshalCapgemini - thank you for your feedback. 

The keyboard focus you see in Xamarin.Forms apps on Android is exactly the same as that which you should see on built-in Android apps. As far as I'm aware, we are not doing anything to influence the keyboard focus to behave otherwise.

What you're describing sounds to me like a custom feature you are seeking. If you could share some screenshots portraying the behavior you described, and clarify what you're referring to with "keyboard focus outlines" and "keyboard focus indicator" so that I can better understand your situation, that would be super helpful :)

**Comment by IeuanWalker** [comment]
@PureWeen There are cases where we need to override the Keyboard with the default focus colour.

For example, android keyboard focus darkens the button, but if you have a dark colour button it's not noticeable enough, so to fix that we override it by adding this to our styles.xml -
```xml
 <item name="android:colorControlHighlight">@color/selected_item_focus</item>
```

Still looking into how to do the same on iOS but this looks promising, just not sure how to use it - https://developer.apple.com/documentation/appkit/nscolor/1532031-keyboardfocusindicatorcolor


I work for the local government and we have legal legislation for our app to be accessible, this was one of the issues they brought up when they audited our app - https://www.gov.uk/government/publications/accessibility-monitoring-of-public-sector-websites-and-mobile-apps-2020-2021/accessibility-monitoring-of-public-sector-websites-and-mobile-apps-2020-2021
